script.js

```diff
@@ -54,83 +65,276 @@ function createEpisodeDescription(tvEpisodeDiv, episode) {
+function createShowDescription(tvShowDiv, show) {
+  const showDescDiv = document.createElement('div');
+  showDescDiv.classList.add('show-description');
+
+  showDescDiv.insertAdjacentHTML('beforeend', show.summary);
+
+  tvShowDiv.appendChild(showDescDiv);
+}
+
+//create show genre
+function createShowGenre(tvShowDiv, show) {
+  const showGenre = document.createElement('div');
+  showGenre.classList.add('show-genre');
+
+  showGenre.innerHTML = `Genres: ${show.genres}`;
+
+  tvShowDiv.appendChild(showGenre);
+}
+
+//create show status
+function createShowStatus(tvShowDiv, show) {
+  const showStatus = document.createElement('div');
+  showStatus.classList.add('show-status');
+
+  showStatus.innerHTML = `Status: ${show.status}`;
+
+  tvShowDiv.appendChild(showStatus);
+}
+
+//create show rating
+function createShowRating(tvShowDiv, show) {
+  const showRating = document.createElement('div');
+  showRating.classList.add('show-rating');
+
+  showRating.innerHTML = `Rated: ${show.rating.average}`;
+
+  tvShowDiv.appendChild(showRating);
+}
+
+//create show runtime
+function createShowRuntime(tvShowDiv, show) {
+  const showRuntime = document.createElement('div');
+  showRuntime.classList.add('show-runtime');
+
+  showRuntime.innerHTML = `Runtime: ${show.runtime}`;
+  tvShowDiv.appendChild(showRuntime);
+}
+
+// present a listing of all shows displaying name, image, summary, genres, status, rating, and runtime
+function makePageForShows() {
+  const rootElem = document.getElementById("root");
+  const showCounter = document.getElementById("showsAndEpisodesCounter");
+  showCounter.textContent = `Got ${allShows.length} show(s)`;
+
+  allShows.forEach((show) => {
+    const tvShowDiv = createTVShowDiv();
+    createShowTitle(tvShowDiv, show);
+
+    //check if medium property is present
+    if (show?.image?.medium != undefined) {
+      createShowImage(tvShowDiv, show);
+    }
+
+    //check if summary property is present
+    if (show.summary != undefined) {
+      createShowDescription(tvShowDiv, show);
+    }
+
+    //check if genre property is present
+    if (show.genre != undefined) {
+      createShowGenre(tvShowDiv, show);
+    }
+
+    //check if status property is present
+    if (show.status != undefined) {
+      createShowStatus(tvShowDiv, show);
+    }
+
+    //check if rating property is present
+    if (show.rating?.average != undefined) {
+      createShowRating(tvShowDiv, show);
+    }
+
+    //check if runtime property is present
+    if (show.runtime != undefined) {
+      createShowRuntime(tvShowDiv, show);
+    }
+
+    rootElem.appendChild(tvShowDiv);
+  });
+}
+
+//------------------------------ SEARCH INPUT EPISODES ----------------------
 //Add live search input
 function search() {
   const input = document.getElementById("search");
   const filter = input.value.toLowerCase();
   const rootElem = document.getElementById("root");
 
-  const allEpisodes = getAllEpisodes();
-  
   const filteredEpisodes = allEpisodes.filter(
```

Comment:
well done for using the `allEpisodes` variable so that you don't have to call the API again!